homeassistant/components/zwave_me/manifest.json

```
@@ -0,0 +1,15 @@
+{
+  "domain": "zwave_me",
+  "name": "Z-Wave.Me",
+  "documentation": "https://github.com/Z-Wave-Me/ha-core",
+  "iot_class": "cloud_polling",
```

It's push based, right?
```suggestion
  "iot_class": "cloud_push",
```